daemon/oci_windows_test.go

```diff
@@ -311,3 +311,97 @@ func setRegistryOpenKeyFunc(t *testing.T, key *dummyRegistryKey, err ...error) f
 		registryOpenKeyFunc = previousRegistryOpenKeyFunc
 	}
 }
+
+func TestSetupWindowsDevices(t *testing.T) {
+	t.Run("it does nothing if there are no devices", func(t *testing.T) {
```

Comment:
More a nit / consistency thing; I don't think we use these kind of descriptions for (sub)tests, and generally use shorter terms (which may be easier to grep and/or use to run only specific sub-tests). I realise this convention is used by other projects and/or testing frameworks though 😅

Reply:
Umm... I copied this style from the other test in this file. Happy to shorten them though, they were a bit awkward to write as they are.

Reply:
Umm... I copied this style from the other test in this file. Happy to shorten them though, they were a bit awkward to write as they are.

Reply:
oh! interesting; should've looked in this file itself ☺️. No worries; it was mostly a nit (and something that stood out, because we generally don't use this format, but apparently we did in this file); I'm fine keeping it as-is